builder/fixtures/buildpacks/unix/has-buildpack-config/bin/finalize

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# vim: set ft=sh
```

Comment:
Probably should remove this `vim: set ft=sh` bit

Reply:
Fixed